config/v1/types_infrastructure.go

```diff
@@ -1002,6 +1002,16 @@ type PowerVSPlatformStatus struct {
 	// Note: Currently only single-zone OCP clusters are supported
 	Zone string `json:"zone"`
 
+	// resourceGroup is the resource group name for new IBMCloud resources created for a cluster.
+	// The resource group specified here will be used by cluster-image-registry-operator to set up a COS Instance in IBMCloud for the cluster registry.
+	// More about resource groups can be found here: https://cloud.ibm.com/docs/account?topic=account-rgs.
+	//
+	// +kubebuilder:validation:Pattern=^[a-zA-Z0-9-_ ]+$
+	// +kubebuilder:validation:MinLength=1
```

Comment:
I think this might cause issues since we haven't got an omitempty tag, which is correct for discoverability, we don't really need this so you can drop it IMO

Reply:
sure, removed min len.